plugin.video.hgtv/resources/lib/scraper.py

```diff
@@ -49,18 +49,23 @@ def getAddonMenu(self,url,ilist):
        except: infoList['Episode'] = 0
        if infoList['Episode'] == 0: continue
        html = self.getRequest(url.rsplit('/',1)[0])
-       try:    plot = re.compile('"og:description" content="(.+?)"',re.DOTALL).search(html).group(1)
-       except: plot = name
-       try:    
-           fanart = re.compile('property="og:image" content="(.+?)"',re.DOTALL).search(html).group(1)
-           fanart = fanart.replace('616.347.jpeg','1280.720.jpeg',1)
-       except: fanart = thumb
+       plot = re.compile('"og:description" content="(.+?)"',re.DOTALL).search(html)
+       if plot != None:
+           plot = plot.group(1)
+       else:
+           plot = name
+       fanart = re.compile('property="og:image" content="(.+?)"',re.DOTALL).search(html)
+       if fanart != None:
+           fanart = fanart.group(1).replace('616.347.jpeg','1280.720.jpeg',1)
+       else:
+           fanart = thumb
        infoList['TVShowTitle'] = name
        infoList['Title']       = name
        infoList['Studio']      = 'HGTV'
        infoList['Plot'] = h.unescape(plot)
+       infoList['mediatype'] = 'tvshows'
```

Comment:
Ok.Wasn't sure. I used this list for content types http://mirrors.kodi.tv/docs/python-docs/15.x-isengard/xbmcplugin.html#-setContent , where are the mediatypes listed?